iroh-net/src/discovery/pkarr.rs

```diff
@@ -21,27 +63,56 @@ use crate::{
     key::SecretKey,
     AddrInfo, Endpoint, NodeId,
 };
+
 #[cfg(feature = "discovery-pkarr-dht")]
 pub mod dht;
 
-/// The pkarr relay run by n0, for production.
+/// The production pkarr relay run by [number 0].
+///
+/// This server is both a pkarr relay server as well as a DNS resolver, see the [module
+/// documentation].  However it does not interact with the Mainline DHT, so is a more
```

Comment:
`iroh-dns-server` *can* interact with the mainline DHT for resolving node ids not found in the local database, see [here](https://github.com/n0-computer/iroh/blob/main/iroh-dns-server/src/store.rs#L66). It does not (yet?) support publishing records to mainline DHT.

 The [config template for production](https://github.com/n0-computer/iroh/blob/main/iroh-dns-server/config.prod.toml#L16) does *not* enable the mainline support by defualt. IIRC we wanted to enable it on `dns.iroh.link` though, I do not know OTOH if that is the case. I would guess @Arqu can verify.

Reply:
Im not sure, lets chat about this on the meeting today.

Reply:
If it is currently not enabled let's leave it documented like this.  I'm not trying to change anything here, merely documenting the current state.